docs/troubleshooting-guide/db-connection.md

```diff
@@ -54,6 +54,8 @@ If you see this error messages in the [logs](./server-logs.md) (**Admin** > **Tr
 2. Go to **Advanced options** > **Additional JDBC connection string options** and add `trustServerCertificate=true`.
 3. Click **Save**.
 
+Also, you might want to check if the driver that Metabase is using to connect to your Data Warehouse supports the version of your product. E.g. before v46, we don't support Microsoft SQL 2022
```

Comment:
```suggestion
The version of Metabase you're using to connect to your data warehouse must support your database's version. For example, Metabase versions older than 46 lack support for Microsoft SQL Server 2022.
```

Is this what we're saying?

And do we have a list of versions for each of the databases that Metabase supports that people could refer to?

Reply:
yup! that's what we want to say. Unfortunately we don't have all lists :(. I just wanted to leave that there for posterity